docs/container.md

```diff
@@ -38,43 +38,56 @@ Next you will need to [Build a Xailient SDK](buildSdk.md) and choose the **X86-6
 <br>
 ### Obtaining a Link to a Xailient SDK
 
-Go to the [MANAGE AI MODEL page](https://console.xailient.com/user/listModel) and locate the model for which you have built the SDK.
+Go to the Console and navigate to [MANAGE AI MODEL page](https://console.xailient.com/user/listModel). For the model you want to deploy, select the SDK you have build for your target platform. 
 
-Click on __Download/Copy SDK__ button for that model. 
+!!! note
+    If you have not build an SDK yet, refer to __Build SDK__ section of the documentation.
+
+Click on the <img src="../img/console/AI Models/Copy.png" height=30 width=30> icon left side of the platform to copy the downlooad link.
 
 <p align="center">
-<img src="../img/console/SDKBuildComplete.png" heigth=100>
+  <img src="../img/console/AI Models/PreTrainedModels-SDKBuilt-copy.png">
 </p>
 
-A link will be copied to your clipboard and a download will begin.
+Click on the target platform for the model to download.
+
+<p align="center">
+  <img src="../img/console/AI Models/PreTrainedModels-SDKBuilt-downlaod.png">
+</p>
 
 <br>
 ## Run the Xailient SDK Container
 
-    `$ docker run -e SDK_LINK="<LINK>" -v <INPUT_IMAGES_DIR>:/input -v <OUTPUT_IMAGES_DIR>:/output xailient/model-inference:1.0.1`
+    $ docker run -e SDK_LINK="<LINK>" -v "<INPUT_IMAGES_DIR>":/input -v "<OUTPUT_IMAGES_DIR>":/output xailient model-inference:1.0.1
```

Comment:
Looks like "xailient/model-inference:1.0.1" was changed to "xailient model-inference:1.0.1" (no slash). I don't think that will do the right thing now.

Reply:
Resolved the issue.